client/components/wordpress-logo/docs/example.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WordPressLogo from 'components/wordpress-logo';
+import Card from 'components/card';
+
+module.exports = React.createClass( {
+	displayName: 'WordPressLogo',
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/wordpress-logo">WordPress Logo</a>
+				</h2>
+				<Card>
```

Comment:
sure :+1:  

Any suggestion on which should be the minimum size for using this component instead of a gridicon?